manifests/deis-router-rc.yaml

```diff
@@ -23,3 +23,7 @@ spec:
         ports:
         - containerPort: 80
           hostPort: 80
+        - containerPort: 443
+          hostPort: 443
+        - containerPort: 2222
```

Comment:
I wouldn't want 2 load balancers - just if we can somehow affect the configuration on *one* LB via multiple Charts :)